include/datatypes.h

```diff
@@ -38,73 +38,173 @@
 #ifndef DATATYPES_H_
 #define DATATYPES_H_
 
-#define DH_KEY_SIZE        20
+#define SAFE_FREE(ptr)     {if (NULL != (ptr)) {free(ptr); (ptr)=NULL;}}
+#define CMK_BLOB_SIZE(x)    (sizeof(ehsm_keyblob_t) + x*sizeof(uint8_t))
+#define EHSM_DATA_SIZE(x)    (sizeof(ehsm_data_t) + x*sizeof(uint8_t))
```

Comment:
rename to:
SIZE_OF_KEYBLOB_T(x)
SIZE_OF_DATA_T(x)

Reply:
It will be change.